Gemfile

```diff
@@ -34,6 +34,7 @@ gem 'rack'
 gem 'rack-utf8_sanitizer'
 gem 'rdoc'
 gem 'rest-client', require: 'rest_client'
+gem 'sass', require: false
```

Comment:
I think this avoids requiring it in production and Sprockets will only require it during precompilation.